gradle.properties

```diff
@@ -1,4 +1,4 @@
-release_version=3.0.1
+release_version=3.0.2
```

Comment:
Please, update the minor version instead. It should be 3.1.0

Reply:
oh, yep